posts/2022-12-05-survey-launch.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Launching the 2022 State of Rust Survey"
+author: The Rust Survey Team
+description: "Hearing from you about the seventh year of Rust"
+---
+
+It's that time again! Time for us to take a look at who the Rust community is composed of, how the Rust project is doing, and how we can improve the Rust programming experience. The Rust Survey Team is pleased to announce our [2022 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses, and establish development priorities for the future.
+
+Completing this survey should take about 5–20 minutes and is anonymous. We will be accepting submissions for the next two weeks (until the 19th of December), and we will write up our findings afterwards on [blog.rust-lang.org]. You can also check out [last year’s results][2021 survey].
```

Comment:
I think it might be helpful to add a loose timeframe of when folks can expect analysis and results. Something like this:

> [...] and we will share our findings on [blog.rust-lang.org] in January 2023.